### ISSUE TITLE
UDPipe model

### ISSUE THREAD

**Issue by matyaskopp** [ufal/ParlaMint-UA#85]
new UD models are out, we have to decide which one we want to use:
https://ufal.mff.cuni.cz/udpipe/2/models#ukrainian-iu-ud-2.15-241121

**Comment by AnnaParla** [comment]
It is possible to test these UD models?

**Comment by matyaskopp** [comment]
> It is possible to test these UD models?

yes, they are available on https://lindat.mff.cuni.cz/services/udpipe/

**Comment by AnnaParla** [comment]
ok

**Comment by matyaskopp** [comment]
consider using [ukrainian-parlamint-ud-2.15-241121](https://ufal.mff.cuni.cz/udpipe/2/models#ukrainian-parlamint-ud-2.15-241121)  see also [Developing a Universal Dependencies Treebank for Ukrainian Parliamentary Speech](https://aclanthology.org/2025.unlp-1.7.pdf)
